src/entry_points/eest_utils.py

```diff
@@ -0,0 +1,87 @@
+"""
+CLI interface for common EEST shortcuts and utilities.
+"""
+
+import argparse
+import os
+import subprocess
+
+from logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+def initialize_repository(args):
+    """
+    Initializes and installs all required packages for the repository:
+        ```
+        python3 -m venv ./venv/
+        source ./venv/bin/activate
+        pip install -e '.[docs,lint,test]'
+        ```
+    """
+    logger.info("Creating a virtual environment: `python3 -m venv ./venv/`")
+    subprocess.run(["python3", "-m", "venv", "./venv/"], check=True)
+    pip_path = os.path.join(".", "venv", "bin", "pip")
+
+    logger.info("Upgrading pip to the latest version: `pip install --upgrade pip`")
+    subprocess.run([pip_path, "install", "--upgrade", "pip"], check=True)
+
+    logger.info("Installing required packages: `pip install -e '.[docs,lint,test]'`")
+    subprocess.run([pip_path, "install", "-e", ".[docs,lint,test]"], check=True)
+
+
+def clean_repository(args):
+    """
+    Cleans the repository of all generated files and directories:
+        ```
+        rm -rf .tox .pytest_cache .mypy_cache venv fixtures
+        ```
+    """
+    items_to_remove = [".tox", ".pytest_cache", ".mypy_cache", "venv", "fixtures"]
+    for item in items_to_remove:
+        if os.path.exists(item):
+            subprocess.run(["rm", "-rf", item], check=True)
```

Comment:
I'd consider using `shutil.rmtree` here to avoid subprocess https://docs.python.org/3/library/shutil.html#shutil.rmtree.